### PR TITLE
feat(apisix): allow extPlugins to be configured

### PR DESCRIPTION
**Pull request by lyondhill**

### CODE REVIEW

File: charts/apisix/README.md [hunk 1]
```diff
@@ -141,6 +141,14 @@ If etcd.enabled is true, set more values of bitnami/etcd helm chart use etcd as
 
 Default enabled plugins. See [configmap template](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/configmap.yaml) for details.
 
+
+### extPlugin parameters
```
Comment (tokers) [inline charts/apisix/README.md:145]:
```suggestion
### external plugin parameters
```

File: charts/apisix/values.yaml [hunk 2]
```diff
@@ -210,13 +210,19 @@ plugins:
...
 stream_plugins:
   - mqtt-proxy
   - ip-restriction
   - limit-conn
 
 pluginAttrs: {}
 
+extPlugin:
+  enabled: false
+  cmd: []
```
Comment (tokers) [inline charts/apisix/values.yaml:224]: Could you give some examples about `cmd`?

File: charts/apisix/README.md [hunk 3]
```diff
@@ -141,6 +141,14 @@ If etcd.enabled is true, set more values of bitnami/etcd helm chart use etcd as
 
 Default enabled plugins. See [configmap template](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/configmap.yaml) for details.
 
+
+### external plugin parameters
+
+| Parameter                       | Description                                                                                                                                                      | Default                     |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| `extPlugin.enabled` | Whether to allow an ext-plugin | `false` |
```
Comment (tokers) [inline charts/apisix/README.md:149]: I prefer to add the hyperlink of external plugins: https://apisix.apache.org/docs/apisix/next/external-plugin/

Comment (lyondhill) [reply charts/apisix/README.md:149]: Happy to add a hyperlink here

**Review by tokers — APPROVED** [review]

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (tokers) [inline _config.yml:1]: What's this?

Comment (tao12345666333) [inline _config.yml:1]: We don't need it

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 4 above)
Comment (lyondhill) [reply _config.yml]: Yah that was added by github. I will clean it.

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**